examples/todomvc/src/components/Footer.js

```diff
@@ -0,0 +1,72 @@
+import React, { PropTypes, Component } from 'react'
+import classnames from 'classnames'
+import { SHOW_ALL, SHOW_COMPLETED, SHOW_ACTIVE } from '../constants/TodoFilters'
+
+const FILTER_TITLES = {
+  [SHOW_ALL]: 'All',
+  [SHOW_ACTIVE]: 'Active',
+  [SHOW_COMPLETED]: 'Completed'
+}
+
+export default class Footer extends Component {
```

Comment:
Suggestion: I think Footer and the call to `Footer` will be simpler if you just pass in the store, and mark it `observer`?